src/Workspaces/CSharpTest/CodeGeneration/SyntaxGeneratorTests.cs

```diff
@@ -1809,6 +1809,20 @@ public void TestUnsafeFieldDeclarationFromSymbol()
 }");
         }
 
+        [Fact]
+        public void TestEnumDeclarationFromSymbol()
+        {
+            VerifySyntax<EnumDeclarationSyntax>(
+                    Generator.Declaration(
+                        _emptyCompilation.GetTypeByMetadataName("System.DateTimeKind")),
+@"public enum DateTimeKind
+{
+    Unspecified = 0,
+    Utc = 1,
+    Local = 2
+}");
+        }
+
         #endregion
```

Comment:
can you add a VB test as well?

Reply:
Sure

Reply:
Done, found out it was returning a SyntaxTree with no members, I also fixed the logic to return the members correctly on VB.